Gemfile

```diff
@@ -1,6 +1,10 @@
 source "http://rubygems.org"
 
+gem 'webmock'
 gemspec
+%w[rspec rspec-core rspec-expectations rspec-mocks rspec-support].each do |lib|
+  gem lib, :git => "git://github.com/rspec/#{lib}.git", :branch => 'master'
+end
```

Comment:
Are we using rspec anywhere? Thought the tests here are running via mocha.

Reply:
Good point, initially I thought test was failing due to missing rspec dependency but like you said, turns out it was mocha so can remove this